apps/easypid/src/features/activity/FunkeActivityScreen.tsx

```diff
@@ -69,7 +69,8 @@ export function FunkeActivityScreen() {
                       subtitle={activity.entityName ?? activity.entityHost}
                       date={new Date(activity.date)}
                       type={activity.type}
-                      credentialId={activity.credentialId}
+                      // FIXME: Handle multiple credentials received in one request
+                      credentialId={activity.type === 'received' ? activity.credentialIds?.[0] : undefined}
```

Comment:
what happens if you delete the cred?

Reply:
It routes to the credential detail screen atm, so it will just toast you that the credential can not be found.